releasenotes/notes/require-cloud-backend-38a74144347c735d.yaml

```
@@ -0,0 +1,6 @@
+---
+deprecations:
+  - |
+    In a future release, ``backend`` will be a required parameter for :class:`qiskit_ibm_runtime.Sampler`, 
+    :class:`qiskit_ibm_runtime.Estimator`, :class:`qiskit_ibm_runtime.Session` and 
+    :class:`qiskit_ibm_runtime.Batch` when using the ``ibm_cloud`` channel.
```

We should be more explicit that `backend` is only required if `session` is not specified. 

```suggestion
    In a future release, ``backend`` will be a required parameter for :class:`qiskit_ibm_runtime.Sampler`, and
    :class:`qiskit_ibm_runtime.Estimator` if ``session`` is not specified,
    even when using the ``ibm_cloud`` channel. 
    
    It will also be a required parameter for 
    :class:`qiskit_ibm_runtime.Session` and 
    :class:`qiskit_ibm_runtime.Batch`.
```